ci/build_docs.sh

```diff
@@ -32,8 +32,15 @@ rapids-print-env
 RAPIDS_DOCS_DIR="$(mktemp -d)"
 export RAPIDS_DOCS_DIR
 
+# Trap ERR so that `EXITCODE` is printed when a command fails and the script
+# exits with error status
 EXITCODE=0
-trap "EXITCODE=1" ERR
+# shellcheck disable=SC2317
+set_exit_code() {
+    EXITCODE=$?
+    rapids-logger "Test failed with error ${EXITCODE}"
```

Comment:
I think of errors as a longer textual description. This is just an exit code.
```suggestion
    rapids-logger "Test failed with exit code ${EXITCODE}"
```